quickstarts/minikube.md

```diff
@@ -1,37 +1,72 @@
-# Starting Minikube
+Minikube provides a local Kubernetes, designed to make it easy to learn and develop for Kubernetes.
+The Kubernetes cluster is started inside a virtual machine, running a Linux kernel and a Docker daemon.
```

Comment:
This is not that easy anymore. By default (if installed) it will use Docker driver.

Reply:
Do you think the following is more representative of the current state of minikube:
```
The Kubernetes cluster is started either inside a virtual machine, a container or on bare-metal, depending on the minikube driver you choose.
```

Reply:
Yeah, I think that works.